core/src/main/java/org/neo4j/ogm/context/GraphEntityMapper.java

```diff
@@ -284,7 +284,7 @@ private void writeProperty(ClassInfo classInfo, Object instance, Property<?, ?>
 
         FieldInfo writer = classInfo.getFieldInfo(property.getKey().toString());
 
-        if (writer == null) {
+        if (writer == null || writer.isComposite()) {
             logger.debug("Unable to find property: {} on class: {} for writing", property.getKey(), classInfo.name());
         } else {
```

Comment:
```suggestion
        if (writer == null) {
            logger.debug("Unable to find property: {} on class: {} for writing", property.getKey(), classInfo.name());
        } else if (writer.isComposite()) {
           logger.info("Property is already handled by a CompositeAttributeConverter"); // or similar wording
```

Reply:
Yes that makes sense. Just FYI, for other properties that are handled by the CompositeAttributeConverter, which do not have the name conflict, the method gets also called and the `writer` is actually `null`. Therefore, the original debug message might also be misleading, but we can probably not differentiate between "somehow we could not find the property" and "property is already handled" in the first if.